src/activities/intro_gravity/Intro_gravity.qml

```diff
@@ -0,0 +1,205 @@
+/* GCompris - intro_gravity.qml
+*
+* Copyright (C) 2015 Siddhesh suthar <siddhesh.it@gmail.com>
+*
+* Authors:
+*   Bruno Coudoin <bruno.coudoin@gcompris.net> and Matilda Bernard (GTK+ version)
+*   Siddhesh suthar <siddhesh.it@gmail.com> (Qt Quick port)
+*
+*   This program is free software; you can redistribute it and/or modify
+*   it under the terms of the GNU General Public License as published by
+*   the Free Software Foundation; either version 3 of the License, or
+*   (at your option) any later version.
+*
+*   This program is distributed in the hope that it will be useful,
+*   but WITHOUT ANY WARRANTY; without even the implied warranty of
+*   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+*   GNU General Public License for more details.
+*
+*   You should have received a copy of the GNU General Public License
+*   along with this program; if not, see <http://www.gnu.org/licenses/>.
+*/
+import QtQuick 2.2
+import QtQuick.Controls 1.3
```

Comment:
Use 'import QtQuick.Controls 1.2' to accept older Qt 5.3 which we use on Android